From linux-usb-devel-bounces@lists.sourceforge.net Thu Oct 11 08:30:03 2007
From: Ortwin Glück <odi@odi.ch>
Date: Thu, 11 Oct 2007 17:29:43 +0200
Subject: USB: Nikon D40X unusual_devs entry
To: linux-usb-devel@lists.sourceforge.net
Message-ID: <470E4167.1000704@odi.ch>

From: Ortwin Glück <odi@odi.ch>

Not surprisingly the Nikon D40X DSC needs the same quirks as the D40,
but it has a separate ID.
See http://bugs.gentoo.org/show_bug.cgi?id=191431

From: Ortwin Glück <odi@odi.ch>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -369,6 +369,13 @@ UNUSUAL_DEV(  0x04b0, 0x0415, 0x0100, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_FIX_CAPACITY),
 
+/* Reported by Shan Destromp (shansan@gmail.com) */
+UNUSUAL_DEV(  0x04b0, 0x0417, 0x0100, 0x0100,
+		"NIKON",
+		"NIKON DSC D40X",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_FIX_CAPACITY),
+
 /* BENQ DC5330
  * Reported by Manuel Fombuena <mfombuena@ya.com> and
  * Frank Copeland <fjc@thingy.apana.org.au> */
